lapce-data/src/config.rs

```diff
@@ -168,7 +168,7 @@ pub struct EditorConfig {
     #[field_names(
         desc = "Set the editor line height. If less than 5.0, line height will be a multiple of the font size."
     )]
-    line_height: f64,
+    pub line_height: f64,
```

Comment:
Doesn't have to be public

Reply:
> Doesn't have to be public

At this case I get a compilation error.

Reply:
What error? I've just built HEAD without any issues

Reply:
> What error? I've just built HEAD without any issues

I don't know. For the first time I got a compilation error. Anyway, now everything is ok!